helpers/rayMath.ts

```diff
@@ -0,0 +1,73 @@
+import BigNumber from "bignumber.js";
```

Comment:
This file is unused?

Reply:
Will be used for wad/ray conversion on tests

Reply:
Okay